### ISSUE TITLE
Running behind a reverse proxy: Cannot resolve dependency

### ISSUE THREAD

**Issue by JonathanHuot** [parcel-bundler/parcel#1163]
**Choose one:** is this a 🐛 bug report or 🙋 feature request?

🙋 feature request

### 🎛 Configuration (.babelrc, package.json, cli command)

```html
<html>
  <body>
    <img src="/images/foobar.jpg"/>
  </body>
</html>
```

```
$ <run api>
$ <run local cdn>
$ npx parcel src/index.html --no-cache
```

I'm running a reverse proxy locally which is dispatching requests depending their URLs:
- `/api/*` => pointing to a xyz API
- `/images/*` => pointing to an image service, store images in memory or in database
- `/dist/*` => pointing to parceljs
- `/*` => pointing to parceljs

### 🤔 Expected Behavior

I expect to have parcel running

### 😯 Current Behavior

Parcel is throwing an error and fail to start: 
Cannot resolve dependency './../../../../../images/foobar.jpg' at '/images/foobar.jpg'

### 💁 Possible Solution

Either: 
- be able to run parceljs without checking every assets URLs,
- add a flag to *parcel* to downgrade severity from errors to warning
- anything ideas ? :-)

I had a look on the changes included in the next release e.g. https://github.com/parcel-bundler/parcel/pull/850 but I think it's a different issue.

### 🔦 Context

I'm trying to use `parceljs` in a micro-services architecture but I can't.

### 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           |1.6.2
| Node             |9.9.0
| npm/Yarn         |5.7.1
| Operating System |

**Comment by DeMoorJasper** [comment]
Update to 1.7 to make sure the resolver hasn't fixed it, as absolute paths got fixed in 1.7

**Comment by JonathanHuot** [comment]
Hi, sorry it's not the main problem. The image doesn't exist in the filesystem because it's provided by a *image service* which is available only at runtime (thru a reverse proxy running locally).

**Comment by DeMoorJasper** [comment]
In that case I'm pretty sure your proxy is configured incorrectly

**Comment by JonathanHuot** [comment]
Yeah, I guess adding extra configuration is reasonable, either in the markup (e.g. `parcel-ignore=1`) or in a config file. As long as it makes this scenario possible... cuz at the moment I am blocked and I'm not able to run `parcel` with the current micro-services architecture :cry: :cry:
